diagrams/sequence-diagrams/README.md

```diff
@@ -0,0 +1,50 @@
+# GOV.UK Forms Sequence Diagrams 
+
+## Introduction
+
+The diagrams in this folder (created using [Mermaid](https://docs.github.com/en/get-started/writing-on-github/working-with-advanced-formatting/creating-diagrams#creating-mermaid-diagrams)) show the interactions between users and GOV.UK Forms, including the 3 applications [forms-admin](https://github.com/alphagov/forms-admin), [forms-api](https://github.com/alphagov/forms-api), and [forms-runner](https://github.com/alphagov/forms-runner).
+
+## High Level Architecture of GOV.UK Forms
+
+```mermaid
+graph TD
+    subgraph users
+        signon_admin((GOV.UK Signon<br />administrator))
+        creator((form creator))
+        content((content designer))
+        user((Person filling in a form))
+        processor((form processor))
+    end
+
+    signon(GOV.UK Signon)
+    govuk(GOV.UK website)
+
+    signon_admin --- signon
+    creator --- signon
+    content --- govuk
```

Comment:
Should the content designer also be using signon?
```suggestion
    content --- govuk
    content --- signon
```

Reply:
They do, but I chose to abstract away how the publishing process works. I don't think there's a benefit in showing the link, plus we'll be moving away from Signon for authentication which will make it even less relevant.